src/bin/tsservices_dev.js

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line node/no-unpublished-import
+import * as ts from "typescript";
+import path from 'path';
+import fs from 'fs';
+
+import { createServicesHost } from 'app/tsservices/serviceshost';
+
+const LIBDTS = fs.readFileSync('src/node_modules/app/tsservices/lib.d.ts').toString();
+const srcname = "datasets.ts";
+const src = `${fs.readFileSync(path.resolve('static/dsl', srcname))}
```

Comment:
I'd prefer having path resolved relatively to this `__dirname` so that should we invoke this file not from root it would still work.

```suggestion
const srcname = 'datasets.ts';
const STATIC_DSL_PATH = path.resolve(__dirname, '../../static/dsl', srcname);
const LIBDTS_PATH = path.resolve(__dirname, '../node_modules/app/tsservices/lib.d.ts');
const LIBDTS = fs.readFileSync(LIBDTS_PATH).toString();
const src = `${fs.readFileSync(STATIC_DSL_PATH, srcname))}
```

(eventually `srcname` could be `SRC_NAME`)